<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_base_inputTable</class>
 <widget class="QWidget" name="Widget_base_inputTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget_base_inputTable
{
border: 1px solid #000000;
background-color: white;
}

#Dialog_other_send_terminal
{
background-color:white;
}

QLineEdit 
{
font-family: '黑体';
font-size: 13px;
}

QComboBox
{
font-family: '黑体';
font-size: 14px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget
{
border: 1px solid #000000;
background-color: white;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>41</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel { 
border: none; 
background-color : rgb(105, 105, 105);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_title">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>24</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel { 
border: none; 
color : rgb(105, 105, 105);
}</string>
            </property>
            <property name="text">
             <string>标题</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButton_selectAll">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
            </property>
            <property name="text">
             <string>全选</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_unSelectAll">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
            </property>
            <property name="text">
             <string>反选</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>758</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_output">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
            </property>
            <property name="text">
             <string>导出</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_input">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
            </property>
            <property name="text">
             <string>导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_del">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_revise">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
            </property>
            <property name="text">
             <string>修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_filter">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
            </property>
            <property name="text">
             <string>筛选</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_add">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
            </property>
            <property name="text">
             <string>新增</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>1161</width>
          <height>551</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">/*设置标题(包括垂直+水平的)*/
QHeaderView
{							
	
	font-size: 15px;				/*字大小*/
	font-family: '黑体';
	/*border: 1px solid #B4B4B4;*/
	border:none;
    /*border-bottom: 2px solid #999999; */		/*下边框颜色*/
    /*background: #9DC6E6;*/				/*背景浅蓝色*/ 
	min-height:30px;
	text-align: center;	       
}


/*设置标题(水平的)*/
QHeaderView::section:horizontal { 					
        border: 1px solid #AFAFAF; 			/*间隔颜色*/
	    border-bottom: 2px solid #999999; 							
        color: black;
        background: #9DC6E6;
}

/*设置鼠标停留状态*/
QHeaderView::section:horizontal:hover 
{			
        color: black;							/*字体白色*/
        background: #82AFC9;				/*背景深蓝色*/
}

/*设置鼠标按下状态*/
QHeaderView::section:horizontal:pressed 
{			
        color: black;
        background: #82AFC9;
		border-bottom: 3px solid #037BFC;				
}


QTableWidget 
{
		font-size: 14px;				/*10榜*/
		font-family: '黑体';				
		color : black;				
        border: 1px solid #000000;		
        background: white;
        gridline-color: #E0E0E0;	
        text-align: center;	
        outline:0px;			/*禁止焦点*/
}

/*设置选中的视图项*/
QTableWidget::item:selected
{				    
		border: 2px solid #0398FC;	
        color : black;				
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_confirm">
        <property name="spacing">
         <number>7</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_confirm">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_confirm">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
